src/middlewares/comment.validation.middleware.js

```diff
@@ -38,9 +64,10 @@ class CommentValidation {
     * @returns { Object } user response as object
     */
   static async deleteCommentValidation(req, res, next) {
-    const id = req.params.commentID;
-    if (!isValide(id)) return response.errorMessage(res, 'Comment id must be a number', 400);
-    const comment = await Queries.findOneRecord(db.comment, { id });
+    const id = parseInt(req.params.commentID, 10);
+    const subjectId = req.params.subjectID;
+    if (!isValide(id)) return response.errorMessage(res, 'Accommodation id and comment id must be a number', 400);
```

Comment:
@shemaeric1234 you've already checked the accommodationID type in the validateSubjectAvailability middleware, so no need to re-do